Lib/test/test_netrc.py

```diff
@@ -1,5 +1,9 @@
-import netrc, os, unittest, sys, textwrap
-from test.support import os_helper
+import netrc
+import os
+import sys
+import textwrap
+import unittest
+from test.support import os_helper, run_unittest
```

Comment:
This causes a `ImportError: cannot import name 'run_unittest' from 'test.support'` (https://github.com/python/cpython/pull/104511#issuecomment-2573202251).